.circleci/real_config.yml

```
@@ -5632,7 +5498,7 @@ workflows:
           name: publish-python-package-rc
           matrix:
             parameters:
-              path: ["harness", "model_hub"]
+              path: ["harness"]
```

I wish people would just use the regular YAML lists for consistency :)
```suggestion
              path:
                - "harness"
```